app/src/app/store/mapRenderSubs.ts

```diff
@@ -61,8 +61,6 @@ export const getRenderSubscriptions = (
       }
 
       const layersToFilter = mapDocument?.child_layer ? CHILD_LAYERS : [];
-
-      if (mapDocument?.child_layer) layersToFilter.push(...CHILD_LAYERS);
```

Comment:
This line was where the max call stack issue seemed to occur. I think (not 100% sure) that this was somehow sometimes it was recursively calling the spread operator and making a huge CHILD_LAYERS reference? Or every time you called this is was adding to CHILD_LAYERS since it is a reference?